### PR TITLE
Fix for imported packages

### PR DESCRIPTION
**Pull request by samsagaz**
Simple Fix for #35

### CODE REVIEW

File: pytest_odoo.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
     odoo = openerp
     odoo_namespace = 'openerp'
 except ImportError:  # Odoo >= 10.0
-    import odoo  # noqa
+    import odoo.tests:wq  # noqa
```
Comment (Jerther) [inline pytest_odoo.py]: Looks like you're using VIM ;)

**Review by Jerther — CHANGES_REQUESTED** [review]

File: pytest_odoo.py (hunk 1 above)
Comment (samsagaz) [reply pytest_odoo.py]: lol, sorry :facepalm:

**Comment by simahawk** [comment]
is this still relevant?

**Comment by simahawk** [comment]
feel free to reopen

**Pull request closed without merging**